OverlayPlugin.Core/MemoryProcessors/Combatant/Common.cs

```diff
@@ -53,7 +53,7 @@ public enum ObjectStatus : byte
          */
     }
 
-    /// <summary>An int offset by 0x104 from the combatant's address that describes the 3D model visibility.</summary>
+    /// <summary>An int offset by 0x114 from the combatant's address that describes the 3D model visibility.</summary>
```

Comment:
```suggestion
    /// <summary>An int that describes the 3D model visibility.</summary>
```

Personally I think we should not duplicate this value here, because it'll just get out of date eventually.